tests/acceptance/features/apiSpaces/quota.feature

```diff
@@ -65,3 +65,24 @@ Feature: State of the quota
     And user "Alice" has uploaded a file inside space "Project Delta" with content "7 bytes" to "test.txt"
     When user "Alice" uploads a file inside space "Project Delta" with content "00011 bytes" to "test.txt" using the WebDAV API
     Then the HTTP status code should be "507"
+
+
+  Scenario Outline: Confirm the relative amount of quota of personal space
+    Given user "Admin" has changed the quota of the personal space of "Alice Hansen" space to "10000"
+    And user "Alice" has uploaded a file from "<file_upload>" to "uploaded.txt" via TUS inside of the space "Alice Hansen" using the WebDAV API
+    When the user "Alice" requests these endpoints with "GET" with basic auth
+      | endpoint    |
+      | <end_point> |
+    Then the HTTP status code should be "200"
+    And the json responded should contain these key and value pairs:
+      | key              | value            |
+      | quota@@@free     | <quota_free>     |
+      | quota@@@used     | <quota_used>     |
+      | quota@@@total    | <quota_total>    |
+      | quota@@@relative | <quota_relative> |
```

Comment:
Can't we use the existing steps? `Then the json responded should contain a space "Alice Hansen" with these key and value pairs:`

Reply:
@roshanlc existing step can be utilized if already there.

Reply:
That method cannot be used as it returns the following response
```json
{
  "ocs": {
    "meta": {
      "status": "ok",
      "statuscode": 100,
      "message": "OK"
    },
    "data": {
      "enabled": "true",
      "quota": {
        "free": 50000000,
        "used": 50000000,
        "total": 100000000,
        "relative": 50
      },
      "email": "alice@example.org",
      "displayname": "Alice Hansen",
      "user-type": "primary",
      "two_factor_auth_enabled": false,
      "last_login": 0
    }
  }
}
```

Is there any pre-existing method for such responses?

Reply:
Ohh! then you need to do `ocs@@@data@@@quota@@@free`

Reply:
Need new step